src/Core/AndroidNative/maui/src/test/java/com/microsoft/maui/PlatformInteropContextDestructionTest.java

```diff
@@ -0,0 +1,145 @@
+package com.microsoft.maui;
+
+import android.app.Activity;
+import android.content.Context;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.Mockito.*;
+
+/**
+ * Unit tests for context destruction checks in PlatformInterop.
+ * Tests the defensive measures to prevent Glide IllegalArgumentException crashes.
+ * 
+ * These tests verify that when contexts are destroyed, the image loading methods
+ * return early and call the callback with false, preventing Glide crashes.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class PlatformInteropContextDestructionTest {
```

Comment:
The test should be in C#, not Java. We don't want to add new tests projects, as that would require AzDO/yaml/CI changes. You didn't do those changes, so it wouldn't run as-is.